README.md

```diff
@@ -44,14 +44,22 @@ We would like your contributions to come as feedbacks, testing, development etc.
 If you are interested in financial donation to the project, or to the developers, you can do so at our [opencollective](https://opencollective.com/kadalu) page. (We like github sponsors too, but its still in waiting list for an org in India).
 
 
+## Helm support
+
+`helm install kadalu https://github.com/kadalu/kadalu/releases/download/<release>/kadalu-helm-chart.tgz`
```

Comment:
Please add `--create-namespace` arg before URL, it's idempotent and will not cause any issues.